app/models/friendship.rb

```diff
@@ -1,4 +1,13 @@
 class Friendship < ApplicationRecord
-  belongs_to :sender, foreign_key: 'sender_id', class_name: 'User'
-  belongs_to :receiver, foreign_key: 'receiver_id', class_name: 'User'
+  after_update(:add_reverse_friendship_pair)
+  belongs_to :invitation_sender, class_name: 'User'
+  belongs_to :invitation_receiver, class_name: 'User'
+
+  def add_reverse_friendship_pair
+    friendship = Friendship.find(id)
+    return unless friendship
+
+    Friendship.create(invitation_sender_id: friendship.invitation_sender_id,
+                      invitation_receiver_id: friendship.invitation_receiver_id, status: true)
+  end
```

Comment:
- [ ] It is good to create a reverse record of friendship once it is approved, but kindly make sure to also reverse sender_id and receiver_id in the new record.
```
eg: #1=> sender_id: 1, receiver_id: 2
      #2=> sender_id: 2, receive_id: 1
```